src/site/content/en/blog/link-prefetch/index.md

```diff
@@ -45,6 +47,10 @@ For example, in a product listing page, you can prefetch the page for the most p
 eBay implemented prefetching for the first five results on a search page to speed up future pages loads and saw a positive impact on conversion rates.
 {% endAside %}
 
+Prefetching entire pages—while costly in terms of bandwidth—can greatly improve most performance metrics. [Time to First Byte (TTFB)](/ttfb/) will often be much lower, as the document request results in a cache hit. Because TTFB will be much lower, any other subsequent time-based metrics will often be much faster, including [Largest Contentful Paint (LCP)](/lcp/) and [First Contentful Paint (FCP)](/fcp/).
+
+[Cumulative Layout Shift (CLS)](/cls/) can also see signficiant improvements, especially in cases where web fonts are necessary to render text. Because the web fonts used by the page will be prefetched along with the HTML document, any font-related layout shifts will be eliminated, as the fonts responsible for the shift won't be network-bound, and will load instantly.
```

Comment:
How often is it the case that a page will use different fonts than the calling page? In my experience that's pretty rare, so not sure this is a great example. So I'd probably cut this line to be honest, if there isn't a better example.

```suggestion
```